pylib/cqlshlib/cql3handling.py

```diff
@@ -1501,6 +1507,38 @@ def alter_type_field_completer(ctxt, cass):
                        ;
 '''
 
+syntax_rules += r'''
+<slaName> ::= <identifier>
+             | <quotedName>
+             | <unreservedKeyword>
+             ;
+
+<createSlaStatement> ::= "CREATE" "SERVICE_LEVEL" ( "IF" "NOT" "EXISTS" )? <slaName>
+                              ( "WITH" <slaProperty> ("AND" <slaProperty>)*)?
+                        ;
+
+<alterSlaStatement> ::= "ALTER" "SERVICE_LEVEL" ("IF" "EXISTS")? <slaName>
+                              ( "WITH" <slaProperty> ("AND" <slaProperty>)*)?
+                       ;
+
+<slaProperty> ::= "WORKLOAD_TYPE" "=" <stringLiteral>
+                 | "TIMEOUT" "=" <wholenumber>
+                 | "SHARES" "=" <wholenumber>
+                 ;
+
+<dropSlaStatement> ::= "DROP" "SERVICE_LEVEL" ("IF" "EXISTS")? <slaName>
+                      ;
+
+<listSlaStatement> ::= "LIST" ("ALL" "ATTACHED")? "SERVICE_LEVELS"
```

Comment:
In the docs I also see different variants: https://enterprise.docs.scylladb.com/stable/using-scylla/workload
-  `LIST SERVICE_LEVEL <name>;`
- `LIST ATTACHED SERVICE_LEVEL OF <role>;`

This grammar also doesn't support `LIST ALL SERVICE_LEVELS`, it requires `ATTACHED`, doesn't it? Version without `ATTACHED` is also mentioned in the docs.

Reply:
@Lorak-mmk 

`LIST ALL SERVICE_LEVELS` is supported:
https://enterprise.docs.scylladb.com/stable/using-scylla/workload-prioritization.html#:~:text=service%20level%20allocations%3A-,LIST%20ALL%20SERVICE_LEVELS%3B,-service_level%20%7C%20shares

Reply:
> @Lorak-mmk
> 
> `LIST ALL SERVICE_LEVELS` is supported: https://enterprise.docs.scylladb.com/stable/using-scylla/workload-prioritization.html#:~:text=service%20level%20allocations%3A-,LIST%20ALL%20SERVICE_LEVELS%3B,-service_level%20%7C%20shares

I meant that it is supported by Scylla, but grammar in this PR does not seem to support it, unless I'm reading it wrong.
Grammar in this PR also seems to not support `LIST SERVICE_LEVEL <name>;` and `LIST ATTACHED SERVICE_LEVEL OF <role>;`, both variants are supported by Scylla Enterprise.

Reply:
@fruch ?

Reply:
ping

Reply:
now it's addressed, and there a unit test to cover it